src/main/docs/guide/ioc/introspection/accessKind.adoc

```diff
@@ -0,0 +1,10 @@
+By default Java introspections treat only JavaBean getters/setters or Java 16 record components as bean properties. You can however define classes with public or package protected fields in Java using the `accessKind` member of the ann:core.annotation.Introspected[] annotation:
```

Comment:
```suggestion
By default Java introspections treat only JavaBean getters/setters or Java record components as bean properties. You can however define classes with public or package protected fields in Java using the `accessKind` member of the ann:core.annotation.Introspected[] annotation:
```

Reply:
specifying that records were added in java 16 is not incorrect